crates/matrix-sdk-crypto/src/olm/group_sessions/inbound.rs

```diff
@@ -55,6 +55,26 @@ use crate::{
     },
 };
 
+/// The trust level required to decrypt an event
+#[derive(Clone, Copy, Debug, Deserialize, Serialize)]
+pub enum TrustRequirement {
+    /// Decrypt events from everyone regardless of trust
+    Untrusted,
+    /// Only decrypt events from cross-signed or legacy devices
```

Comment:
People won't know what legacy means, myself included :sweat_smile:. Could you define that.